app/views/facility_orders/index.html.haml

```diff
@@ -34,6 +37,14 @@
               %td.centered= check_box_tag("order_detail_ids[]", od.id, false, {class: "toggle", id: nil })
               %td.centered= link_to od.order_id, facility_order_path(current_facility, od.order)
               %td.centered= link_to od.id, manage_order_detail_path(od), class: "manage-order-detail"
+              - if cross_core_order_view_enabled
+                %td
+                  = od.order.created_by_user.full_name
+                  - if od.order.cross_core_project.present?
+                    - if od.order.cross_core_project.facility == current_facility
+                      %i.fa.fa-building{ data: { toggle: "tooltip" }, title: t("views.facility_orders.index.cross_core_icon_tooltip", facility: od.order.cross_core_project.facility.abbreviation)}
+                    - else
+                      %i.fa.fa-users{ data: { toggle: "tooltip" }, title: t("views.facility_orders.index.cross_core_icon_tooltip", facility: od.order.cross_core_project.facility.abbreviation)}
```

Comment:
I might pull this into a partial so we can re-use it.  should we have a new column for the icons/facility abbreviations?

Reply:
Ticket said to use icons, but today Jeff said to use a new column. Maybe we can ask at the Consortium meeting?

Reply:
Move icon to new column called Facility. Next to the icon, show the facility abbreviation (check whether it needs to be truncated to 6 chars). Keep icon on hover behavior